### DISCUSSION TITLE
ww3_tp2.14

### DISCUSSION THREAD

**Discussion by Agoni00** [NOAA-EMC/WW3#709] [Q&A]
Hello, I have a problem when I run the ww3_tp2.14 command (./bin/run_test -c OASIS -c Altix -s OASOCM -f -o netcdf -w work_OASOCM -p mpirun -n 4 .. /model ww3_tp2.14), there will be a problem as shown in the picture below, how should I solve this problem? Thank you very much for your answer.
![1652757559(1)](https://user-images.githubusercontent.com/92639723/168722034-c6d6b78a-39bd-49cd-af78-1a57296d56bb.jpg)

**Comment by mickaelaccensi** [comment]
Hi,
Have you solved your error ?  Have you ran the ftp_from_ww3.sh to download all the necessary input files ? you can also look into the log files to understand why the toy model has not been compiled correctly. Also please the latest version of ww3 because we have updated many files for ww3_tp2.14
